### PR TITLE
Add document for funnel charts

### PR DESCRIPTION
**Pull request by haotran-avaiga**
Addresses #302

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/funnel.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
+# Funnel charts
+
+Funnel charts represent a series of decreasing values as shrinking stacked bars.
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/funnel.md]: charts represent a series => charts represent series ?

File: docs/manuals/gui/viselements/charts/funnel.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+| [*x*](../chart.md#p-x)            | Volume values for stages  | For the `funnel` type.  |
+| [*y*](../chart.md#p-y)            | Stage names  | For the `funnel` type.  |
+| [*values*](../chart.md#p-values)  | Volume values for stages  | For the `funnelarea` type.  |
+| [*text*](../chart.md#p-text)      | Stage names  | For the `funnelarea` type.  |
+| [*marker*](../chart.md#p-marker)  | dictionary  | `color` can be used to indicate the color of the box representing a stage.<br/>`line` provides the parameters for styling the box frames. |
+| [*options*](../chart.md#p-options)  | dictionary  | `connector` can be used to customize the graphical properties of the lines connecting the boxes. |
+
+## Examples
+
+### Simple funnel chart {data-source="gui:doc/examples/charts/funnel-simple"}
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/funnel.md:31]: data-source has no extension ?

File: docs/manuals/gui/viselements/charts/funnel.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+
+```py
+data = {
+    "Types":  ["Website visit", "Downloads", "Prospects", "Invoice sent", "Closed"],
+    "Visits": [13873, 10533, 5443, 2703, 908]
+}
+
+marker = {
+    # Boxes are filled with a blue gradient color
+    "color": ["hsl(210,50%,50%)", "hsl(210,60%,60%)", "hsl(210,70%,70%)", "hsl(210,80%,80%)", "hsl(210,90%,90%)"],
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/funnel.md:150]: color could also point the a column in the data set

File: docs/manuals/gui/viselements/charts/funnel.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+
+The *names* variable holds all the trace names, so they appear properly in the chart
+legend.
+
+!!! example "Page content"
+
+    === "Markdown"
+
+        ```
+        <|{data}|type=funnel|chart|x={x}|y=Types|name={names}|>
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/funnel.md]: does this work ?
I would have thought {data}|chart|type=...

**Review by FredLL-Avaiga — APPROVED** [review]
Corel Auto-preserve dir might supererogatory ?

File: docs/manuals/gui/viselements/charts/funnel.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/funnel.md]: True!

File: docs/manuals/gui/viselements/charts/funnel.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/funnel.md:150]: Yes. That would be a more general indication, but yes.

Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/funnel.md:150]: Done in the 'key properties section'.

File: docs/manuals/gui/viselements/charts/funnel.md (hunk 4 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/funnel.md]: Whao. Indeed this shouldn't work!
I messed up my copy/paste I guess. Thx!

File: docs/manuals/gui/viselements/charts/funnel.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/funnel.md:31]: No need.
For the time being, it will be a raw .py file. In the future, a directory?

**Pull request merged**